com.zsmartsystems.zigbee.dongle.telegesis/src/main/java/com/zsmartsystems/zigbee/dongle/telegesis/ZigBeeDongleTelegesis.java

```diff
@@ -342,7 +342,7 @@
     private final int defaultS10 = 0x56A9;
 
     /**
-     * Map used to correlate the Telegesis sequence IDs with the APS counter we use to correlate messages in the stack
+     * Map used to correlate the Telegesis sequence IDs with the Transaction ID we use to correlate messages in the stack
```

Comment:
We actually use transaction ID as a key in that map and in equivalents in the other dongles code. This comment mislead me a bit so correcting